.github/workflows/mingw.yml

```diff
@@ -36,13 +36,13 @@ jobs:
 
       - name: Cache MinGW
         id: cache-mingw
-        uses: actions/cache@v1.0.3
+        uses: actions/cache@v1.1.2
         with:
           path: D:\a\_temp\msys
           key: windows-mingw
 
       - name: Cache Build
-        uses: actions/cache@v1.0.3
+        uses: actions/cache@v1.1.2
         with:
           path: build
           key: mingw-${{ matrix.EVENT_MATRIX }}
```

Comment:
does not have prefix for cache, let's use one unique prefix across all builds (so everywhere in this file)

Reply:
have changed to `mingw-autotools-${{ matrix.EVENT_MATRIX }}`